actor-testkit-typed/src/main/scala/org/apache/pekko/actor/testkit/typed/CapturedLogEvent.scala

```diff
@@ -15,14 +15,14 @@ package org.apache.pekko.actor.testkit.typed
 
 import java.util.Optional
 
-import org.slf4j.Marker
-import org.slf4j.event.Level
-
 import org.apache.pekko
 import pekko.annotation.InternalApi
 import pekko.util.OptionConverters._
 import pekko.util.OptionVal
 
+import org.slf4j.Marker
```

Comment:
@He-Pin there are some formatting changes that are unrelated to this PR (like this one), would it be a big deal to revert them?